### ISSUE TITLE
CT: TypeError: localRefToMovableShape is null

### ISSUE THREAD

**Issue by jbphet** [phetsims/area-builder#128]
There's a new CT failure on Area Builder.  I haven't touched the code in here for eons, so it is most likely a change to common code.

**Comment by jbphet** [comment]
I've done some investigation and can now duplicate manually.  Here are the steps:

1. Load the sim, go to the "Games" screen
2. Choose level 4
3. Start dragging shapes from the creator area at the bottom onto the shape on the grid area
4. Continue dragging new instances until the problem occurs

**Comment by jbphet** [comment]
The problem is occurring when the shape creator node is set to invisible.  The drag listener function, and specifically the `start` function, is being altered in some way.  There have been some recent changes to `DragListener`, so I'll ask the developer who made them and see if he has any insights.

**Comment by jbphet** [comment]
After looking at the commit logs for `DragListener` I talked with @jessegreenberg and then @jonathanolson, and learned that the drag listeners now have their input interrupted when the associated `Node` goes invisible, but the client can opt out of this.  I tested the "opt out" change and it seemed to work, so I've now committed it.  I'll keep an eye on CT.

For what it's worth, this code is using an old "toolbox" pattern that we may want to eventually replace, but it didn't seem worth it to do so today.
